Makefile

```diff
@@ -386,7 +384,6 @@ ${ISTIO_OUT}/archive: istioctl-all LICENSE README.md istio.VERSION install/updat
 	cp -r tools ${ISTIO_OUT}/archive
 	install/updateVersion.sh -c "$(ISTIO_DOCKER_HUB),$(VERSION)" \
                                  -x "$(ISTIO_DOCKER_HUB),$(VERSION)" -p "$(ISTIO_DOCKER_HUB),$(VERSION)" \
-                                 -i "$(ISTIO_URL)/$(ISTIO_URL_ISTIOCTL)" \
```

Comment:
who/what is going to set that url then?

Reply:
We should remove or cut down istio.VERSION, the only place it is actually used is in vm mesh expansion and there we are using PILOT_DEB_URL and thats it. This is a partial cleanup, nobody is using URL and it is not being set anywhere.

Reply:
I would think we need to either remove it or keep it working ? not in between?